package.json

```diff
@@ -15,7 +15,7 @@
   "module": "dist/index.module.js",
   "jsnext:main": "dist/index.es.js",
   "engines": {
-    "node": ">=10"
```

Comment:
The big thing! node 10 isn't even in maintenance LTS anymore.
https://nodejs.org/en/about/releases/